internal/resources/policy/label_selector_requirement_schema.go

```diff
@@ -80,3 +84,82 @@ func flattenLabelSelectorRequirement(labelSelectorRequirement *policymodel.K8sIo
 
 	return flattenLabelSelectorRequirement
 }
+
+func ValidateSpecLabelSelectorRequirement(ctx context.Context, diff *schema.ResourceDiff, i interface{}) error {
+	value, ok := diff.GetOk(specKey)
+	if !ok {
+		return fmt.Errorf("spec: %v is not valid: minimum one valid spec block is required", value)
+	}
+
+	data, _ := value.([]interface{})
+
+	if len(data) == 0 || data[0] == nil {
+		return fmt.Errorf("spec data: %v is not valid: minimum one valid spec block is required", data)
+	}
+
+	specData := data[0].(map[string]interface{})
+
+	v, ok := specData[NamespaceSelectorKey]
+	if !ok {
+		return fmt.Errorf("namespace_selector: %v is not valid: minimum one valid namespace_selector block is required", v)
+	}
+
+	v1, ok := v.([]interface{})
+	if !ok || len(v1) == 0 || v1[0] == nil {
+		return fmt.Errorf("namespace_selector data: %v is not valid: minimum one valid namespace_selector block is required", v1)
+	}
+
+	namespaceSelectorData, _ := v1[0].(map[string]interface{})
+
+	v2, ok := namespaceSelectorData[MatchExpressionsKey]
+	if !ok {
+		return fmt.Errorf("match expressions: %v is not valid: minimum one valid match expressions block is required", v2)
+	}
+
+	vs, ok := v2.([]interface{})
+	if !ok {
+		return fmt.Errorf("type of match expressions block data: %v is not valid", vs)
+	}
+
+	errStrings := make([]string, 0)
+
+	for _, raw := range vs {
+		if raw != nil {
```

Comment:
flip the condition check here. 
if raw == nil {
continue
}

Reply:
Done